addons/web/static/tests/core/domain_selector_tests.js

```diff
@@ -607,4 +610,25 @@ QUnit.module("Components", (hooks) => {
         await editInput(target, ".o_domain_leaf_value_input", `["b"]`);
         assert.strictEqual(comp.value, `[("state", "in", ["b"])]`);
     });
+
+    QUnit.module("DomainField");
```

Comment:
There is `domain_field_test` for testing `DomainField` and I've just noticed that we already have a test that goes well with your fix but it does not fail without your fix...
Don't know why :thinking: 
You can remove the test as it's already tested, sorry to annoy you with that.

Reply:
Ok, found why, `debug` param is not set in the mount function in tests.
Not a fix for this PR.

Reply:
I assume you're referring to the test `domain field: handle false domain as []`. I don't know why it doesn't fail without this change. Enabling debug mode with `patchWithCleanup(odoo, { debug: true })` doesn't work either.

However, if my test fails without this change and works well with the change, shouldn't we keep it as well?